### PR TITLE
QC script for ADTTE dataset.

### PR DESCRIPTION
**Pull request by robertdevine**
@laxamanaj For the km plot, @SHAESEN2 and Pilot1 Team rmarkdown should be sufficient including system timestamp at pre-submission run. Thanks again.

### CODE REVIEW

**Comment by laxamanaj** [comment]
Thanks so much for this quick update, @robertdevine .  Do you mind restyling and push updates again?

`styler::style_file("qc/qc_adtte.R")`

File: qc/qc_adtte.R [hunk 1]
```diff
@@ -0,0 +1,13 @@
+###########################################################################
+#' developers : Robert Devine/
```
Comment (SHAESEN2) [inline qc/qc_adtte.R:2]: the '/' can be removed

**Review by SHAESEN2 — COMMENTED** [review]
Small comment

File: qc/qc_adtte.R (hunk 1 above)
Comment (robertdevine) [reply qc/qc_adtte.R]: Thanks @laxamanaj and @SHAESEN2.

1. > styler::style_file("qc/qc_adtte.R")
Styling  1  files:
 qc/qc_adtte.R ℹ 
────────────────────────────────────────
Status	Count	Legend 
✔ 	0	File unchanged.
ℹ 	1	File changed.
✖ 	0	Styling threw an error.
────────────────────────────────────────
Please review the changes carefully!

2. '/' removed

**Comment by robertdevine** [comment]
By way of observation, all the other Pilot3 qc/qc_*.R scripts utilize '/' in each case. 

###########################################################################
#' developers : Kangjie Zhang/

###########################################################################
#' developers : Thomas Neitmann/

Did we stumble upon a bug? @laxamanaj, @SHAESEN2 ?

**Review by laxamanaj — APPROVED** [review]
Thanks for removing the `/`.  I think this should be used when multiple developers.  As you're the only one, we can remove.  For other scripts that have this after single developer, it makes sense to remove.  I'm happy to do this in branch so that they're consistent.

**Pull request merged**